integrations/azure/steps/complete-ort-run.yml

```diff
@@ -0,0 +1,101 @@
+# Copyright (C) 2020 Bosch.IO GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+# This file contains a complete ort run: analyze - scan - evaluate - report
+# The run will be done in one job building the Dockerfile beforehand
```

Comment:
Rephrase this sentence "The run will be done in one job building the Dockerfile beforehand". What do you mean here?